[2025-05-31T23:36:09.330+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: entrenamiento_modelos_regresion.entrenar_y_promover manual__2025-05-31T23:30:59.666437+00:00 [queued]>
[2025-05-31T23:36:09.343+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: entrenamiento_modelos_regresion.entrenar_y_promover manual__2025-05-31T23:30:59.666437+00:00 [queued]>
[2025-05-31T23:36:09.343+0000] {taskinstance.py:2170} INFO - Starting attempt 2 of 2
[2025-05-31T23:36:09.358+0000] {taskinstance.py:2191} INFO - Executing <Task(_PythonDecoratedOperator): entrenar_y_promover> on 2025-05-31 23:30:59.666437+00:00
[2025-05-31T23:36:09.362+0000] {standard_task_runner.py:60} INFO - Started process 341 to run task
[2025-05-31T23:36:09.366+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'entrenamiento_modelos_regresion', 'entrenar_y_promover', 'manual__2025-05-31T23:30:59.666437+00:00', '--job-id', '21', '--raw', '--subdir', 'DAGS_FOLDER/entrenamiento_modelos_regresion_dag.py', '--cfg-path', '/tmp/tmp3chlxu7s']
[2025-05-31T23:36:09.368+0000] {standard_task_runner.py:88} INFO - Job 21: Subtask entrenar_y_promover
[2025-05-31T23:36:09.431+0000] {task_command.py:423} INFO - Running <TaskInstance: entrenamiento_modelos_regresion.entrenar_y_promover manual__2025-05-31T23:30:59.666437+00:00 [running]> on host 993869572b99
[2025-05-31T23:36:09.532+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='fede' AIRFLOW_CTX_DAG_ID='entrenamiento_modelos_regresion' AIRFLOW_CTX_TASK_ID='entrenar_y_promover' AIRFLOW_CTX_EXECUTION_DATE='2025-05-31T23:30:59.666437+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-31T23:30:59.666437+00:00'
[2025-05-31T23:36:10.813+0000] {logging_mixin.py:188} WARNING - /opt/***/dags/entrenamiento_modelos_regresion_dag.py:56 SettingWithCopyWarning:
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
[2025-05-31T23:36:10.824+0000] {logging_mixin.py:188} WARNING - /opt/***/dags/entrenamiento_modelos_regresion_dag.py:59 SettingWithCopyWarning:
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
[2025-05-31T23:36:10.826+0000] {logging_mixin.py:188} WARNING - /opt/***/dags/entrenamiento_modelos_regresion_dag.py:61 SettingWithCopyWarning:
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
[2025-05-31T23:36:10.827+0000] {logging_mixin.py:188} WARNING - /opt/***/dags/entrenamiento_modelos_regresion_dag.py:63 SettingWithCopyWarning:
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
[2025-05-31T23:36:10.905+0000] {logging_mixin.py:188} WARNING - 2025/05/31 23:36:10 WARNING mlflow.utils.git_utils: Failed to import Git (the Git executable is probably not on your PATH), so Git SHA is not available. Error: Failed to initialize: Bad git executable.
The git executable must be specified in one of the following ways:
    - be included in your $PATH
    - be set via $GIT_PYTHON_GIT_EXECUTABLE
    - explicitly set via git.refresh(<full-path-to-git-executable>)

All git commands will error until this is rectified.

This initial message can be silenced or aggravated in the future by setting the
$GIT_PYTHON_REFRESH environment variable. Use one of the following values:
    - quiet|q|silence|s|silent|none|n|0: for no message or exception
    - warn|w|warning|log|l|1: for a warning message (logging level CRITICAL, displayed by default)
    - error|e|exception|raise|r|2: for a raised exception

Example:
    export GIT_PYTHON_REFRESH=quiet
[2025-05-31T23:36:14.049+0000] {logging_mixin.py:188} WARNING - 2025/05/31 23:36:14 WARNING mlflow.models.model: Model logged without a signature and input example. Please set `input_example` parameter when logging the model to auto infer the model signature.
[2025-05-31T23:36:14.114+0000] {logging_mixin.py:188} WARNING - 2025/05/31 23:36:14 INFO mlflow.tracking._tracking_service.client: üèÉ View run RandomForest at: http://mlflow:5000/#/experiments/1/runs/e4cc191033174ee781dcd3057fb85868.
[2025-05-31T23:36:14.114+0000] {logging_mixin.py:188} WARNING - 2025/05/31 23:36:14 INFO mlflow.tracking._tracking_service.client: üß™ View experiment at: http://mlflow:5000/#/experiments/1.
[2025-05-31T23:36:16.744+0000] {logging_mixin.py:188} WARNING - 2025/05/31 23:36:16 WARNING mlflow.models.model: Model logged without a signature and input example. Please set `input_example` parameter when logging the model to auto infer the model signature.
[2025-05-31T23:36:16.824+0000] {logging_mixin.py:188} WARNING - 2025/05/31 23:36:16 INFO mlflow.tracking._tracking_service.client: üèÉ View run CatBoost at: http://mlflow:5000/#/experiments/1/runs/1e5789ddea8c4607b4c08e0ce40867a3.
[2025-05-31T23:36:16.825+0000] {logging_mixin.py:188} WARNING - 2025/05/31 23:36:16 INFO mlflow.tracking._tracking_service.client: üß™ View experiment at: http://mlflow:5000/#/experiments/1.
[2025-05-31T23:36:19.123+0000] {logging_mixin.py:188} WARNING - 2025/05/31 23:36:19 WARNING mlflow.models.model: Model logged without a signature and input example. Please set `input_example` parameter when logging the model to auto infer the model signature.
[2025-05-31T23:36:19.173+0000] {logging_mixin.py:188} WARNING - 2025/05/31 23:36:19 INFO mlflow.tracking._tracking_service.client: üèÉ View run SVR at: http://mlflow:5000/#/experiments/1/runs/98643d0d3ca84ed5aaf86be17e40037f.
[2025-05-31T23:36:19.174+0000] {logging_mixin.py:188} WARNING - 2025/05/31 23:36:19 INFO mlflow.tracking._tracking_service.client: üß™ View experiment at: http://mlflow:5000/#/experiments/1.
[2025-05-31T23:36:19.235+0000] {logging_mixin.py:188} WARNING - Registered model 'mejor_modelo_orquestado' already exists. Creating a new version of this model...
[2025-05-31T23:36:19.287+0000] {logging_mixin.py:188} WARNING - 2025/05/31 23:36:19 INFO mlflow.store.model_registry.abstract_store: Waiting up to 300 seconds for model version to finish creation. Model name: mejor_modelo_orquestado, version 2
[2025-05-31T23:36:19.287+0000] {logging_mixin.py:188} WARNING - Created version '2' of model 'mejor_modelo_orquestado'.
[2025-05-31T23:36:19.289+0000] {logging_mixin.py:188} WARNING - /opt/***/dags/entrenamiento_modelos_regresion_dag.py:99 FutureWarning: ``mlflow.tracking.client.MlflowClient.get_latest_versions`` is deprecated since 2.9.0. Model registry stages will be removed in a future major release. To learn more about the deprecation of model registry stages, see our migration guide here: https://mlflow.org/docs/latest/model-registry.html#migrating-from-stages
[2025-05-31T23:36:19.308+0000] {logging_mixin.py:188} WARNING - /opt/***/dags/entrenamiento_modelos_regresion_dag.py:109 FutureWarning: ``mlflow.tracking.client.MlflowClient.transition_model_version_stage`` is deprecated since 2.9.0. Model registry stages will be removed in a future major release. To learn more about the deprecation of model registry stages, see our migration guide here: https://mlflow.org/docs/latest/model-registry.html#migrating-from-stages
[2025-05-31T23:36:19.385+0000] {logging_mixin.py:188} WARNING - /opt/***/dags/entrenamiento_modelos_regresion_dag.py:102 FutureWarning: ``mlflow.tracking.client.MlflowClient.transition_model_version_stage`` is deprecated since 2.9.0. Model registry stages will be removed in a future major release. To learn more about the deprecation of model registry stages, see our migration guide here: https://mlflow.org/docs/latest/model-registry.html#migrating-from-stages
[2025-05-31T23:36:19.443+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-05-31T23:36:19.463+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=entrenamiento_modelos_regresion, task_id=entrenar_y_promover, execution_date=20250531T233059, start_date=20250531T233609, end_date=20250531T233619
[2025-05-31T23:36:19.549+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-05-31T23:36:19.569+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
